src/modlunky2/ui/trackers/label.py

```diff
@@ -0,0 +1,213 @@
+from collections import defaultdict
+from dataclasses import dataclass
+from enum import Enum
+from typing import Optional, Set
+
+
+@dataclass
```

Comment:
Why is this a dataclass when it still has a manual constructor?

Reply:
Good point. I added `@dataclass` while debugging a test and hadn't found `__post_init__`